### PR TITLE
[Enhancement] Update 

### PR DESCRIPTION
**Pull request by tr1xem**
Made it work with current vpn book servers
vpnstop fixed
Added IPV6 mask(better security
Added wrong password fallback
misc things(check commits)
you can contact me on discord or instagram or telegram : tr1x_em

### CODE REVIEW

**Comment by tr1xem** [comment]
Manual password entry dont work for now 
Local fix change the password correct from the website in the .vpnbook/login.conf and then select servers

**Comment by tr1xem** [comment]
Fixed Manual passwords and clean up a bit

File: vpnbook [hunk 1]
```diff
@@ -212,60 +272,62 @@ then
...
-	# Login to sudo	
+	
 	sudo -v
 	
 	# Start the connection and if whatever error occurs, then connect to the first available server
 	__connect_to_first_available ${var}
 	
 	# Check if the given interface is up and running
-	__check_if_up ${vpnbook_if_name}
+	# __check_if_up ${vpnbook_if_name}
```
Comment (HiMyNameIsIlNano) [inline vpnbook:322]: I think this line might be removed

File: vpnbook [hunk 2]
```diff
@@ -75,86 +77,144 @@ __check_requirements(){
...
+                return 1  # Failure, interface didn't come up
+            fi
+         
+        else
+            echo "Failed to establish Vpn Tunnel to ${server}"
+        fi
+    else
+        echo "No server specified. Trying default servers..."
+       
+        for i in pl134 de20 us1 uk205 ca149 fr200; do
```
Comment (HiMyNameIsIlNano) [inline vpnbook]: What is your take if we rename this variable to "l_server" or "server_i"?

File: vpnbook [hunk 3]
```diff
@@ -39,30 +39,32 @@ __read_property(){
...
-	if [ "${group}" == "vpnstarter" ]; then
-		echo "The user is part of the vpnstarter Group"
-	else
-		echo "The user is not part of the vpnstarter Group. Please Fix the issue first and then retry."
-		exit 1
+    mv ${vpnbook_folder}/${base_name}-${u_country}.${ext} ${vpnbook_folder}/${u_country}
 	fi
 }
 
+#__is_user_allowed_to_vpn(){
```
Comment (HiMyNameIsIlNano) [inline vpnbook:58]: I think this whole function might be removed

**Review by HiMyNameIsIlNano — COMMENTED** [review]
Thanks for taking the time to tidy up this script and making it better

File: vpnbook (hunk 3 above)
Comment (tr1xem) [reply vpnbook:58]: Yup was planning to do so 
Kept it because i thought you would need it maybe?

File: vpnbook (hunk 1 above)
Comment (tr1xem) [reply vpnbook:322]: Ig its required to check if the connection is really established or its just gimick
to remove it ig we need to add a check with in manual password entry function

File: vpnbook (hunk 2 above)
Comment (tr1xem) [reply vpnbook]: Sorry Didnt understood that

Comment (HiMyNameIsIlNano) [reply vpnbook]: This variable is named _i_ which works great for loops with integers but in this case we iterate over a list of server names. My suggestion is to use a more suitable name like "server_i" or "l_server" for this variable.

P.S: I am not a fan of prefixes to discriminate between "global" variables and "local" variables (e.g. l_) so I would rather use "server_i" but given that this script has no real coding convention I don't mind using "l_server".

File: vpnbook (hunk 3 above)
Comment (HiMyNameIsIlNano) [reply vpnbook:58]: You can remove it. If I need it I will revert the change.

File: vpnbook (hunk 2 above)
Comment (tr1xem) [reply vpnbook]: Did that

File: vpnbook (hunk 3 above)
Comment (tr1xem) [reply vpnbook:58]: Done

**Comment by tr1xem** [comment]
Did the modification you said

**Pull request merged**